C2CRIBuildDir/projects/C2C-RI/src/RIGUI/log4j2.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration packages="org.fhwa.c2cri.logger" status="TRACE" monitorInterval="30" dest="log4j_status.log">
+	<Appenders>
+		<RollingFile name="C2CRIOutput" fileName="C2CRIOutput.log" append="false" filePattern="C2CRIOutput-%i.dat">
+			<RIXMLLayout />
+			<Policies>
+				<SizeBasedTriggeringPolicy size="100MB" />
+			</Policies>
+			<DefaultRolloverStrategy max="2"/>
+		</RollingFile>
+		<File name="C2CRIDebugFile" bufferedIO="true" fileName="C2CRIDebug.log" append="false">
+			<RIXMLLayout />
+		</File>
+	</Appenders>
+	
+	<Loggers>		
+		<Root level="INFO">
```

Comment:
Comparing to the above log4j.properties, the root logger has both ALL and STDOUT loggers. I am assuming ALL is the console logger? Do we want an ALL logger as well if we run in IDE to see the printed logs?

Reply:
ALL is the level filter for the appender. In log4j2 level filters are a property of the Logger, not the appender. So maybe we should change the Root logger level to ALL. 

STDOUT is just the name (a confusing name) they gave to the appender. It didn't actually write to the console or stdout.